src/content/decentralization/index.md

```diff
@@ -0,0 +1,329 @@
+- Ethereum does not associate your activities with your personal identity (because it doesn't know who you are).
+- Brute-force "identity theft" crimes can be extremely difficult because account credentials are cryptographically protected.
+- You can issue transactions instantly from anywhere with an internet connection.
+- You can check how everything works — and even participate directly in improving it.
+
+### Cons {#cons-of-ethereum}
+
+- Nobody but the recipient can help if you send your funds to the wrong address.
+- You are responsible for account security and privacy practices: for keeping your [private keys](https://ethereum.org/en/glossary/#private-key) secure and keeping your personal information separate from your online activities.
+- You will probably have to learn new information and behaviors.
+- It is your responsibility to identify and comply with applicable laws.
+- It can be difficult to know which contracts, information sources, and apps you should trust.
+
+It basically comes down to whether controlling your money is worth your attention — whether obtaining a higher degree of freedom is worth learning something new. In the end, it's up to you to decide. 
+
+However, know that there are a lot of smart people who are dedicating themselves to making this technology easier for you to use. There is less need for confirmation, problem-solving, and configuration every day. If a decentralized system is simply too technical for you right now, check it out again soon. It's getting simpler every day, from a user's perspective.
+
+It also bears mentioning that you can be one of those smart people working on Ethereum. Perhaps the biggest pro-Ethereum argument out there is that your contributions matter. 
+
+If you use a smart contract or transact on Ethereum, you're doing something to support the entire network. If you give feedback to a developer, asking for new features or reporting issues, that's even better. You can even contribute directly by documenting, translating, coding, or starting your own project.
+
+### Basic Ethereum safety {#safety-ethereum}
+
+Non-technical? Don't worry. The decentralized financial systems of the future will probably be easier to use than centralized finance is today. In the meantime, please take some basic steps to keep your funds safe when you're starting out on Ethereum:
+
+- [Learn about how wallets work](https://ethereum.org/en/wallets/)
+- Confirm (before interacting with a smart contract) that your wallet or exchange account can interact with smart contracts
+- Triple-check destination addresses, even if you're sending things to yourself or to exchanges
+- Send small test transactions before sending big amounts
+- Understand the difference between pseudonymous and private (most blockchains are public, pseudonymous records)
+- Above all else, do not share private keys or mnemonic phrases with others. Not even if they're trying to help. 
```

Comment:
This could use some additional emphasis (if not it's own page eventually) in my opinion. Also, can imagine not everyone knows what a "mnemonic" is, but will understand something like "12 or 24-words"..
One thought:
> Above all else, **backup and do not share private keys or mnemonic phrases (12 or 24-words) with others.** Not even if they're trying to help. They are the master key to your funds and are for your eyes only. These should **only** ever be typed **directly into your wallet**, nowhere else.
```
Above all else, **backup and do not share private keys or mnemonic phrases (12 or 24-words) with others.** Not even if they're trying to help. They are the master key to your funds and are for your eyes only. These should **only** ever be typed **directly into your wallet**, nowhere else.
```